dockstore-webservice/src/main/resources/openapi3/openapi.yaml

```diff
@@ -1841,7 +1841,7 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/Aliasable'
+                $ref: '#/components/schemas/WorkflowVersion'
```

Comment:
Have you merged/rebased from develop recently? I thought this had been fixed by Gary (I just verified the ticket).